portfolio/src/main/webapp/script.js

```diff
@@ -115,15 +122,27 @@ xhttp.onreadystatechange = function() {
       textArea.name = 'text-input';
       textArea.required = true;
 
+      const imgText = document.createElement('p');
+      imgText.innerText = 'Upload an image:';
+
+      const imgInput = document.createElement('input');
+      imgInput.type = 'file';
+      imgInput.name = 'image';
+      imgInput.setAttribute('class', 'btn btn-outline-secondary');
+
       const breakElement = document.createElement('br');
+      const breakElement2 = document.createElement('br');
 
       const button = document.createElement('input');
       button.setAttribute('class', 'btn btn-outline-secondary');
       button.setAttribute('type', 'submit');
 
       form.appendChild(feedbackPrompt);
       form.appendChild(textArea);
+      form.appendChild(imgText);
+      form.appendChild(imgInput);
       form.appendChild(breakElement);
+      form.appendChild(breakElement2);
```

Comment:
Why do we want two consecutive break elements?

Reply:
It's just a style preference for the page. (You also cannot repeat the same break element)